nion/swift/Panel.py

```diff
@@ -420,27 +541,20 @@ def _repaint(self, drawing_context: DrawingContext.DrawingContext) -> None:
                     drawing_context.stroke()
 
             if self.__display_close_control:
-                with drawing_context.saver():
-                    drawing_context.begin_path()
-                    close_box_left = canvas_size.width - (20 - 7)
-                    close_box_right = canvas_size.width - (20 - 13)
-                    close_box_top = canvas_size.height // 2 - 3
-                    close_box_bottom = canvas_size.height // 2 + 3
-                    drawing_context.move_to(close_box_left, close_box_top)
-                    drawing_context.line_to(close_box_right, close_box_bottom)
-                    drawing_context.move_to(close_box_left, close_box_bottom)
-                    drawing_context.line_to(close_box_right, close_box_top)
-                    drawing_context.line_width = 1.5
-                    drawing_context.line_cap = "round"
-                    drawing_context.stroke_style = self.__control_style
-                    drawing_context.stroke()
+                self._draw_controls_windows(drawing_context)
+                # match platform.system():
+                #    case "Darwin": #  MAC
+                #        self._draw_close_control_mac(drawing_context)
+                #    case "Linux":
+                #        self._draw_close_control_linux(drawing_context)
+                #    case _: #  Default to Windows
+                #        self._draw_controls_windows(drawing_context)
+
+
+
+
+
```

Comment:
The `match` is only available in Python 3.10, which is why `mypy` is failing with the cryptic error message. Just use an `if...elif...` construct until we officially deprecate Python 3.9 in the CI.

Reply:
Ah so it was that causing the weird mypy error, and not the package.  I didn't realise that match was added in recently, but sure I can fix that to prep for adding in per platform styling.  It also explains some of the other areas of code that had massive nested if/else statements instead of a single match.
As you say, maybe this is a good reason to consider deprecating the 3.9 build to tidy up those.